restapi/src/main/java/io/stargate/web/docsapi/dao/Paginator.java

```diff
@@ -29,12 +27,15 @@
 
   public Paginator(DataStore dataStore, String pageState, int pageSize, int dbPageSize) {
     this.dataStore = dataStore;
-    docPageSize = Math.max(1, pageSize);
-    if (docPageSize > DocumentDB.MAX_PAGE_SIZE) {
```

Comment:
should we keep the upper `pageSize` validation as well (besides the lower bound)?